modules/rostests/rosautotest/main.cpp

```diff
@@ -139,6 +148,23 @@ wmain(int argc, wchar_t* argv[])
             WineTest.Run();
         }
 
+        /* Clear the stringstream */
+        ss.str("");
+        ss.clear();
+
+        /* Show the beginning time again */
+        ss << "[ROSAUTOTEST] System uptime at start was " << setprecision(2) << fixed;
+        ss << ((float)TestStartTime / 1000) << " seconds" << endl;
+
+        /* Show the time now so that we can see how long the tests took */
+        TestEndTime = GetTickCount();
+        ss << endl
+           << "[ROSAUTOTEST] System uptime at end was " << setprecision(2) << fixed;
+        ss << ((float)TestEndTime / 1000) << " seconds" << endl;
+        ss << "[ROSAUTOTEST] Duration was " << (((float)TestEndTime / 1000) - ((float)TestStartTime / 1000)) / 60;
```

Comment:
```suggestion
        ss << "[ROSAUTOTEST] Duration was " << (((float)TestEndTime - (float)TestStartTime) / 1000) / 60;
```
to keep some precision

Reply:
Changed. Thanks.